tests/api/test_server.py

```diff
@@ -141,3 +141,8 @@ def test_request_azure_with_multi_query(client):
         assert test_query["date"] in image["date"]
         assert test_query["virt"] in image["virt"]
     assert response.status_code == 200
+
+
+def test_request_health(client):
```

Comment:
Could you add a failing test case as well, using a client in which no proper image data is provided?